### PR TITLE
Update workshop from initial feedback

### PR DESCRIPTION
**Pull request by timrafay**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/020_event_engine_setup/26_config_workshop_req.ee.md [hunk 1]
```diff
@@ -56,4 +56,29 @@ the EKS IAM authentication, so we will disable it and rely on the IAM role inste
    If the IAM role is not valid, <span style="color: red;">**DO NOT PROCEED**</span>. Go back and confirm the steps on this page.
    {{% /notice %}}
 
+
+1. Copy and run (paste with **Ctrl+P** or **CMD+P**) the commands below.
```
Comment (parkand1) [inline content/020_event_engine_setup/26_config_workshop_req.ee.md]: Can you throw this under content/030_self_guided_setup/36_config_workshop_req.md? the EE track will have all of this bootstrapped to the Cloud9 environment.

File: content/040_modules/1_setup.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Cloud credentials provide privileges to programmatically interact with your Amaz
 
 - Login to the Rafay Console with the credentials provided and click on Infrastructure
 
-- Select "Cloud Credentials", Click on "New Credential" and provide a unique name "aws-workshop-x" where x aligns with your Rafay user.
+- Select "Cloud Credentials", Click on "New Credential" and provide a unique name "aws-cloudcredential-xx" where xx aligns with your Rafay user.
 
 ![Create Cloud Credential](/040_modules/img/part1/cloud_credential_create.png)
```
Comment (parkand1) [inline content/040_modules/1_setup.md]: For all static images can you please put these under static/images? It's easier to keep track of all static assets this way. Example:(/040_modules/img/part1/cloud_credential_create.png) should be  (/images/cloud_credential_create.png)

File: content/020_event_engine_setup/26_config_workshop_req.ee.md (hunk 1 above)
Comment (timrafay) [reply content/020_event_engine_setup/26_config_workshop_req.ee.md]: The kubectl setup appears to already be in the self guided setup.  Am I missing something?

Comment (parkand1) [reply content/020_event_engine_setup/26_config_workshop_req.ee.md]: This PR has it added to the ee.md file which is specifically for the event engine track

**Review by parkand1 — CHANGES_REQUESTED** [review]
Left a few comments.

**Pull request merged**